src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/integration/WorkingDayIntTest.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.integration
```

Comment:
This replaced the WorkingDayServiceInt test - because WorkingDayServiceInt was meant to be an int test but was bypassing the controller  and going straight into the service.